run/test/specs/utils/open_app.ts

```diff
@@ -28,13 +32,70 @@ export type SupportedPlatformsType = "android" | "ios";
 /* ******************Command to run Appium Server: *************************************
 ./node_modules/.bin/appium server --use-drivers=uiautomator2,xcuitest --port 8110 --use-plugins=execute-driver --allow-cors
 */
+// Basic test environment is 3 devices (device1, device2, device3) and 2 users (userA, userB)
+// Device 1 and 3 are linked devicesby userA
+export const createBasicTestEnvironment = async (
```

Comment:
do we still need this?

Reply:
This was the starting idea of fixtures, is that still something we want to do? I know its not completely correct in what you were wanting.. but it was a start?